src/box/field_def.c

```diff
@@ -233,6 +333,12 @@ field_type_by_name(const char *name, size_t len)
 					       field_type_MAX);
 	if (field_type != field_type_MAX)
 		return field_type;
+	/*
+	 * `uint64' is missed in `field_type_strs', because it is an alias for
+	 * `unsigned'.
+	 */
+	if (len == strlen("uint64") && strncasecmp(name, "uint64", len) == 0)
+		return FIELD_TYPE_UINT64;
```

Comment:
Why make `uint64` an alias? Can't we have it as a separate type but allow conversion to/from `unsigned` without checking values?

Reply:
Done (I tried to avoid duplicate entries in `enum field_type`).